### PR TITLE
Meeting summary

### PR DESCRIPTION
**Pull request by ag12**
#What 
Meeting summary.

#Why 
So we can use it as a reference. 😈

### CODE REVIEW

File: meeting/14-12-18.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+### Long Term goal:
+- Take more responsibility and manual test our code. (A friendly reminder)
```
Comment (robinsalehjan) [inline meeting/14-12-18.md]: As in testing changes/new features for regressions/bugs more?

Comment (ag12) [reply meeting/14-12-18.md]: Exactly 🥇

Comment (bstien) [reply meeting/14-12-18.md]: I'm updating the reference notes now to include this :)

Comment (ag12) [reply meeting/14-12-18.md]: thanks 🤛

File: meeting/14-12-18.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+### Short Term goal: (After Christmas release)
+- Try out our new Cocoapod strategy. (Not commit the Pod folder)
+  Next meeting we will discuss how this is working for us.
+
```
Comment (nickfalk) [inline meeting/14-12-18.md:8]: Fun fact: the "new Cocoapod strategy" is actually the _old_ Cocoapod strategy. 😛

**Review by nickfalk — APPROVED** [review]

File: meeting/14-12-18.md (hunk 2 above)
Comment (ag12) [reply meeting/14-12-18.md:8]: 😍 
Let's see what happens. The New Old Cocopod Strategy. #tnocs

**Comment by ag12** [comment]
Thanks for your help @bstien. Could not say it better myself.

**Pull request merged**